fluss-connectors/fluss-connector-flink/src/test/java/com/alibaba/fluss/connector/flink/catalog/FlinkCatalogTest.java

```diff
@@ -130,7 +129,7 @@ void beforeEach() throws Exception {
         } catch (CatalogException e) {
             // the auto partitioned manager may create the db zk node
             // in an another thread, so if exception is NodeExistsException, just ignore
-            if (!ExceptionUtils.findThrowable(e, KeeperException.NodeExistsException.class)
+            if (!ExceptionUtils.findThrowableWithMessage(e, "KeeperException$NodeExistsException")
```

Comment:
Could you extract this into a separate hotfix pull request? This seems quite unstable in these days. https://github.com/alibaba/fluss/actions/runs/13626469089/job/38084662575